.changeset/clean-needles-taste.md

```diff
@@ -0,0 +1,51 @@
+---
+"@igloo-ui/action-menu": patch
+"@igloo-ui/alert": patch
+"@igloo-ui/area-chart": patch
+"@igloo-ui/avatar": patch
+"@igloo-ui/bar-chart": patch
+"@igloo-ui/breadcrumb": patch
+"@igloo-ui/button": patch
+"@igloo-ui/button-group": patch
+"@igloo-ui/card": patch
+"@igloo-ui/carousel": patch
+"@igloo-ui/checkbox": patch
+"@igloo-ui/color": patch
+"@igloo-ui/color-picker": patch
+"@igloo-ui/combobox": patch
+"@igloo-ui/datepicker": patch
+"@igloo-ui/dialog": patch
+"@igloo-ui/disclosure": patch
+"@igloo-ui/dropdown": patch
+"@igloo-ui/ellipsis": patch
+"@igloo-ui/filter": patch
+"@igloo-ui/form-group": patch
+"@igloo-ui/helper-text": patch
+"@igloo-ui/hyperlink": patch
+"@igloo-ui/icon-button": patch
+"@igloo-ui/input": patch
+"@igloo-ui/list": patch
+"@igloo-ui/metric": patch
+"@igloo-ui/modal": patch
+"@igloo-ui/option-button": patch
+"@igloo-ui/pager": patch
+"@igloo-ui/pie-chart": patch
+"@igloo-ui/popover": patch
+"@igloo-ui/progress-bar": patch
+"@igloo-ui/radio": patch
+"@igloo-ui/select": patch
+"@igloo-ui/stacked-bar": patch
+"@igloo-ui/stepper": patch
+"@igloo-ui/tabs": patch
+"@igloo-ui/tag": patch
+"@igloo-ui/tag-picker": patch
+"@igloo-ui/text-editor": patch
+"@igloo-ui/textarea": patch
+"@igloo-ui/toaster": patch
+"@igloo-ui/toggle": patch
+"@igloo-ui/tooltip": patch
+"@igloo-ui/vertical-bar-chart": patch
+"docs": patch
+---
+
+Updated input icon and character count colors. Also updated hopper token versions and igloo icon versions.
```

Comment:
i think the best practice here would be to create 2 changeset

texteditor with the message: Updated input icon and character count colors
all other pacakges: updated hopper token versions and igloo icon versions.

Reply:
ok I will created two changesets, thanks